src/components/List.js

```diff
@@ -11,10 +11,15 @@ type Grid = {|
   column: number,
 |};
 
+// More info to be added, and probably should be importing type from somewhere else.
```

Comment:
Good call @fragosti , what's the convention for where we should define types?

Is this file--List.js--just for SubmissionInfo? Or should other types go in here as well? Like, is it a "list" of types or was it named with the SubmissionInfo in mind? I'm a little confused by the naming convention. Where should the User type be defined? A new file?

Reply:
In general we'll just be defining data with types. For types that are defining Props to a React component that's easy -- in the component definition file. There are other types like User that are a bit of an open question because those will match with the schema you defined in the backend. This will get a bit tricky because you actually don't want to just redefine those types on the client, since GraphQL allows us to just query the fields we want (if we did that, all the fields in our types would have to be optional). So short answer is I don't know, we'll figure it out. 
I think the most likely solution will be that we have a few User types, with various fields, and they'll typically correspond to a query (so like MinimalUser, FullUser, ProfileUser etc...). In that case we can put them all in a types.js file or just keep them next to their query like we do here :P 

Yes the List.js name is confusing we should fix that. It was initially going to be a wrapper around antd's "List" but then just made it our SubmissionList. Agree that things can be renamed. A lot of things need some polish :)